wordle-web/src/components/LetterBase.vue

```diff
@@ -1,6 +1,6 @@
 <template>
-  <v-btn :color="props.color">
-    {{ props.char }}
+  <v-btn :color="color" class="removeMin">
```

Comment:
Add gradient to guesses blocks and keys (with CSS) ❌
I'm not sure how they wanted us to implement this and use our themes' colors. I ended up just using a gradient and not using the themes for the keys.

Reply:
My dilemma was the 'make it look like the wordle app' task, felt like gradients conflicted with that so I wasn't sure exactly which task to favor. Maybe I'll put a subtle gradient in the non-guessed letters on the gameboard.

Reply:
@Malcolm-Shepherd I just made a commit where I'm using css variables generated by vue from the theme to implement gradients using theme colors, in case you wanna see how it can be done.